cores/esp8266/xtruntime-frames.h

```diff
@@ -0,0 +1,162 @@
+/*  xtruntime-frames.h  -  exception stack frames for single-threaded run-time  */
```

Comment:
Is this header used anywhere?

Reply:
It is used by cores/esp8266/exc-c-wrapper-handler.S.

Is there a better way to organize this? Should this and its friends be moved into a subdirectory?